index.html

```diff
@@ -236,7 +248,7 @@ <h2>Language</h2>
                         </li>
                         <li>
                             <div class="lang-sec">
-                                <img src="icons/rectangle.png" width="18px" height="18px" alt="" />
+                                <img src="images/rhombus.png" width="18px" height="18px" alt="" />
                                 <h2>Frameworks</h2>
```

Comment:
- [ ] Kindly check the three symbols beside language, framework, and skill respectively. Each of them carries different symbol on the Figma design while your design carries the same symbol for all. Kindly attend to that.

### Your Design
![your-lang-frame-skill](https://user-images.githubusercontent.com/26597432/236642908-24d791fe-5b08-4e67-96bd-933e2cbec3d4.png)

### Figma Design
![temp-lang-frame-skill](https://user-images.githubusercontent.com/26597432/236642918-0203e8a5-ad93-4307-9903-fd4361beae44.png)